### ISSUE TITLE
2025-04-15, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#141]
# Today's Arxiv Papers

(1) [xVerify: Efficient Answer Verifier for Reasoning Model Evaluations](http://arxiv.org/abs/2504.10481v1), 2025-04-14

(2) [InternVL3: Exploring Advanced Training and Test-Time Recipes for Open-Source Multimodal Models](http://arxiv.org/abs/2504.10479v1), 2025-04-14

(3) [MIEB: Massive Image Embedding Benchmark](http://arxiv.org/abs/2504.10471v1), 2025-04-14

(4) [Pixel-SAIL: Single Transformer For Pixel-Grounded Understanding](http://arxiv.org/abs/2504.10465v1), 2025-04-14

(5) [The Scalability of Simplicity: Empirical Analysis of Vision-Language Learning with a Single Transformer](http://arxiv.org/abs/2504.10462v1), 2025-04-14

(6) [GUI-R1 : A Generalist R1-Style Vision-Language Action Model For GUI Agents](http://arxiv.org/abs/2504.10458v1), 2025-04-14

(7) [M1: Towards Scalable Test-Time Compute with Mamba Reasoning Models](http://arxiv.org/abs/2504.10449v1), 2025-04-14

(8) [Multimodal Long Video Modeling Based on Temporal Dynamic Context](http://arxiv.org/abs/2504.10443v1), 2025-04-14

(9) [LLM Can be a Dangerous Persuader: Empirical Study of Persuasion Safety in Large Language Models](http://arxiv.org/abs/2504.10430v1), 2025-04-14

(10) [Can We Edit LLMs for Long-Tail Biomedical Knowledge?](http://arxiv.org/abs/2504.10421v1), 2025-04-14

(11) [Unchecked and Overlooked: Addressing the Checkbox Blind Spot in Large Language Models with CheckboxQA](http://arxiv.org/abs/2504.10419v1), 2025-04-14

(12) [CliniChat: A Multi-Source Knowledge-Driven Framework for Clinical Interview Dialogue Reconstruction and Evaluation](http://arxiv.org/abs/2504.10418v1), 2025-04-14

(13) [LLM-SRBench: A New Benchmark for Scientific Equation Discovery with Large Language Models](http://arxiv.org/abs/2504.10415v1), 2025-04-14

(14) [HUMOTO: A 4D Dataset of Mocap Human Object Interactions](http://arxiv.org/abs/2504.10414v1), 2025-04-14

(15) [Performance of Large Language Models in Supporting Medical Diagnosis and Treatment](http://arxiv.org/abs/2504.10405v1), 2025-04-14

(16) [Can LLMs Assist Expert Elicitation for Probabilistic Causal Modeling?](http://arxiv.org/abs/2504.10397v1), 2025-04-14

(17) [LLM-driven Constrained Copy Generation through Iterative Refinement](http://arxiv.org/abs/2504.10391v1), 2025-04-14

(18) [SymRTLO: Enhancing RTL Code Optimization with LLMs and Neuron-Inspired Symbolic Reasoning](http://arxiv.org/abs/2504.10369v1), 2025-04-14

(19) [S1-Bench: A Simple Benchmark for Evaluating System 1 Thinking Capability of Large Reasoning Models](http://arxiv.org/abs/2504.10368v1), 2025-04-14

(20) [FingER: Content Aware Fine-grained Evaluation with Reasoning for AI-Generated Videos](http://arxiv.org/abs/2504.10358v1), 2025-04-14

(21) [MultiLoKo: a multilingual local knowledge benchmark for LLMs spanning 31 languages](http://arxiv.org/abs/2504.10356v1), 2025-04-14

(22) [VisualPuzzles: Decoupling Multimodal Reasoning Evaluation from Domain Knowledge](http://arxiv.org/abs/2504.10342v1), 2025-04-14

(23) [Forecasting from Clinical Textual Time Series: Adaptations of the Encoder and Decoder Language Model Families](http://arxiv.org/abs/2504.10340v1), 2025-04-14

(24) [Heimdall: test-time scaling on the generative verification](http://arxiv.org/abs/2504.10337v1), 2025-04-14

(25) [MorphTok: Morphologically Grounded Tokenization for Indian Languages](http://arxiv.org/abs/2504.10335v1), 2025-04-14

(26) [AlayaDB: The Data Foundation for Efficient and Effective Long-context LLM Inference](http://arxiv.org/abs/2504.10326v1), 2025-04-14

(27) [AutoStyle-TTS: Retrieval-Augmented Generation based Automatic Style Matching Text-to-Speech Synthesis](http://arxiv.org/abs/2504.10309v1), 2025-04-14

(28) [Characterizing LLM-driven Social Network: The Chirper.ai Case](http://arxiv.org/abs/2504.10286v1), 2025-04-14

(29) [Can LLMs Generate Tabular Summaries of Science Papers? Rethinking the Evaluation Protocol](http://arxiv.org/abs/2504.10284v1), 2025-04-14

(30) [Zero-shot Autonomous Microscopy for Scalable and Intelligent Characterization of 2D Materials](http://arxiv.org/abs/2504.10281v1), 2025-04-14

(31) [XY-Cut++: Advanced Layout Ordering via Hierarchical Mask Mechanism on a Novel Benchmark](http://arxiv.org/abs/2504.10258v1), 2025-04-14

(32) [GNN-ACLP: Graph Neural Networks based Analog Circuit Link Prediction](http://arxiv.org/abs/2504.10240v1), 2025-04-14

(33) [Probing then Editing Response Personality of Large Language Models](http://arxiv.org/abs/2504.10227v1), 2025-04-14

(34) [PRM-BAS: Enhancing Multimodal Reasoning through PRM-guided Beam Annealing Search](http://arxiv.org/abs/2504.10222v1), 2025-04-14

(35) [Can Competition Enhance the Proficiency of Agents Powered by Large Language Models in the Realm of News-driven Time Series Forecasting?](http://arxiv.org/abs/2504.10210v1), 2025-04-14

(36) [From Prompting to Alignment: A Generative Framework for Query Recommendation](http://arxiv.org/abs/2504.10208v1), 2025-04-14

(37) [DioR: Adaptive Cognitive Detection and Contextual Retrieval Optimization for Dynamic Retrieval-Augmented Generation](http://arxiv.org/abs/2504.10198v1), 2025-04-14

(38) [Localized Cultural Knowledge is Conserved and Controllable in Large Language Models](http://arxiv.org/abs/2504.10191v1), 2025-04-14

(39) [Deep Reasoning Translation via Reinforcement Learning](http://arxiv.org/abs/2504.10187v1), 2025-04-14

(40) [LLM Unlearning Reveals a Stronger-Than-Expected Coreset Effect in Current Benchmarks](http://arxiv.org/abs/2504.10185v1), 2025-04-14

(41) [The Future of MLLM Prompting is Adaptive: A Comprehensive Experimental Evaluation of Prompt Engineering Methods for Robust Multimodal Performance](http://arxiv.org/abs/2504.10179v1), 2025-04-14

(42) [HalluSearch at SemEval-2025 Task 3: A Search-Enhanced RAG Pipeline for Hallucination Detection](http://arxiv.org/abs/2504.10168v1), 2025-04-14

(43) [C-FAITH: A Chinese Fine-Grained Benchmark for Automated Hallucination Evaluation](http://arxiv.org/abs/2504.10167v1), 2025-04-14

(44) [Fact-Checking with Contextual Narratives: Leveraging Retrieval-Augmented LLMs for Social Media Analysis](http://arxiv.org/abs/2504.10166v1), 2025-04-14

(45) [MT-R1-Zero: Advancing LLM-based Machine Translation via R1-Zero-like Reinforcement Learning](http://arxiv.org/abs/2504.10160v1), 2025-04-14

(46) [COUNTS: Benchmarking Object Detectors and Multimodal Large Language Models under Distribution Shifts](http://arxiv.org/abs/2504.10158v1), 2025-04-14

(47) [SocioVerse: A World Model for Social Simulation Powered by LLM Agents and A Pool of 10 Million Real-World Users](http://arxiv.org/abs/2504.10157v1), 2025-04-14

(48) [HistLLM: A Unified Framework for LLM-Based Multimodal Recommendation with User History Encoding and Compression](http://arxiv.org/abs/2504.10150v1), 2025-04-14

(49) [A Survey of Personalization: From RAG to Agent](http://arxiv.org/abs/2504.10147v1), 2025-04-14

(50) [Benchmarking Practices in LLM-driven Offensive Security: Testbeds, Metrics, and Experiment Design](http://arxiv.org/abs/2504.10112v1), 2025-04-14

(51) [Enhancing LLM-based Recommendation through Semantic-Aligned Collaborative Knowledge](http://arxiv.org/abs/2504.10107v1), 2025-04-14

(52) [The Human Visual System Can Inspire New Interaction Paradigms for LLMs](http://arxiv.org/abs/2504.10101v1), 2025-04-14

(53) [CameraBench: Benchmarking Visual Reasoning in MLLMs via Photography](http://arxiv.org/abs/2504.10090v1), 2025-04-14

(54) [Towards Quantifying Commonsense Reasoning with Mechanistic Insights](http://arxiv.org/abs/2504.10077v1), 2025-04-14

(55) [MMKB-RAG: A Multi-Modal Knowledge-Based Retrieval-Augmented Generation Framework](http://arxiv.org/abs/2504.10074v1), 2025-04-14

(56) [Mavors: Multi-granularity Video Representation for Multimodal Large Language Model](http://arxiv.org/abs/2504.10068v1), 2025-04-14

(57) [Hallucination Detection in LLMs via Topological Divergence on Attention Graphs](http://arxiv.org/abs/2504.10063v1), 2025-04-14

(58) [Emotional Strain and Frustration in LLM Interactions in Software Engineering](http://arxiv.org/abs/2504.10050v1), 2025-04-14

(59) [CodeRAG: Supportive Code Retrieval on Bigraph for Real-World Code Generation](http://arxiv.org/abs/2504.10046v1), 2025-04-14

(60) [CHARM: Calibrating Reward Models With Chatbot Arena Scores](http://arxiv.org/abs/2504.10045v1), 2025-04-14

(61) [DataMosaic: Explainable and Verifiable Multi-Modal Data Analytics through Extract-Reason-Verify](http://arxiv.org/abs/2504.10036v1), 2025-04-14

(62) [EmbodiedAgent: A Scalable Hierarchical Approach to Overcome Practical Challenge in Multi-Robot Control](http://arxiv.org/abs/2504.10030v1), 2025-04-14

(63) [The Mirage of Performance Gains: Why Contrastive Decoding Fails to Address Multimodal Hallucination](http://arxiv.org/abs/2504.10020v1), 2025-04-14

(64) [Training LLMs on HPC Systems: Best Practices from the OpenGPT-X Project](http://arxiv.org/abs/2504.10013v1), 2025-04-14

(65) [Do We Really Need Curated Malicious Data for Safety Alignment in Multi-modal Large Language Models?](http://arxiv.org/abs/2504.10000v1), 2025-04-14

(66) [Enhancing Multi-task Learning Capability of Medical Generalist Foundation Model via Image-centric Multi-annotation Data](http://arxiv.org/abs/2504.09967v1), 2025-04-14

(67) [Privacy Meets Explainability: Managing Confidential Data and Transparency Policies in LLM-Empowered Science](http://arxiv.org/abs/2504.09961v1), 2025-04-14

(68) [C-MTCSD: A Chinese Multi-Turn Conversational Stance Detection Dataset](http://arxiv.org/abs/2504.09958v1), 2025-04-14

(69) [Omni-Dish: Photorealistic and Faithful Image Generation and Editing for Arbitrary Chinese Dishes](http://arxiv.org/abs/2504.09948v1), 2025-04-14

(70) [Assessing Judging Bias in Large Reasoning Models: An Empirical Study](http://arxiv.org/abs/2504.09946v1), 2025-04-14

(71) [KeepKV: Eliminating Output Perturbation in KV Cache Compression for Efficient LLMs Inference](http://arxiv.org/abs/2504.09936v1), 2025-04-14

(72) [Constrained Auto-Regressive Decoding Constrains Generative Retrieval](http://dx.doi.org/10.1145/3726302.3729934), 2025-04-14

(73) [FUSION: Fully Integration of Vision-Language Representations for Deep Cross-Modal Understanding](http://arxiv.org/abs/2504.09925v1), 2025-04-14

(74) [Guiding Reasoning in Small Language Models with LLM Assistance](http://arxiv.org/abs/2504.09923v1), 2025-04-14

(75) [Learning to Erase Private Knowledge from Multi-Documents for Retrieval-Augmented Large Language Models](http://arxiv.org/abs/2504.09910v1), 2025-04-14

(76) [Refining Financial Consumer Complaints through Multi-Scale Model Interaction](http://arxiv.org/abs/2504.09903v1), 2025-04-14

(77) [TAMP: Token-Adaptive Layerwise Pruning in Multimodal Large Language Models](http://arxiv.org/abs/2504.09897v1), 2025-04-14

(78) [Learning from Reference Answers: Versatile Language Model Alignment without Binary Human Preference Data](http://arxiv.org/abs/2504.09895v1), 2025-04-14

(79) [LangPert: Detecting and Handling Task-level Perturbations for Robust Object Rearrangement](http://arxiv.org/abs/2504.09893v1), 2025-04-14

(80) [Investigating Syntactic Biases in Multilingual Transformers with RC Attachment Ambiguities in Italian and English](http://arxiv.org/abs/2504.09886v1), 2025-04-14

(81) [Ember: A Compiler for Efficient Embedding Operations on Decoupled Access-Execute Architectures](http://arxiv.org/abs/2504.09870v1), 2025-04-14

(82) [RadarLLM: Empowering Large Language Models to Understand Human Motion from Millimeter-wave Point Cloud Sequence](http://arxiv.org/abs/2504.09862v1), 2025-04-14

(83) [EthosGPT: Mapping Human Value Diversity to Advance Sustainable Development Goals (SDGs)](http://arxiv.org/abs/2504.09861v1), 2025-04-14

(84) [SUMART: SUMmARizing Translation from Wordy to Concise Expression](http://dx.doi.org/10.1109/VRW62533.2024.00119), 2025-04-14

(85) [Reasoning Models Can Be Effective Without Thinking](http://arxiv.org/abs/2504.09858v1), 2025-04-14

(86) [Working with Large Language Models to Enhance Messaging Effectiveness for Vaccine Confidence](http://arxiv.org/abs/2504.09857v1), 2025-04-14

(87) [PestMA: LLM-based Multi-Agent System for Informed Pest Management](http://arxiv.org/abs/2504.09855v1), 2025-04-14

(88) [A Survey of Large Language Model-Powered Spatial Intelligence Across Scales: Advances in Embodied Agents, Smart Cities, and Earth Science](http://arxiv.org/abs/2504.09848v1), 2025-04-14

(89) [StruPhantom: Evolutionary Injection Attacks on Black-Box Tabular Agents Powered by Large Language Models](http://arxiv.org/abs/2504.09841v1), 2025-04-14

(90) [RAKG:Document-level Retrieval Augmented Knowledge Graph Construction](http://arxiv.org/abs/2504.09823v1), 2025-04-14

(91) [Transferable text data distillation by trajectory matching](http://arxiv.org/abs/2504.09818v1), 2025-04-14

(92) [Augmented Relevance Datasets with Fine-Tuned Small LLMs](http://arxiv.org/abs/2504.09816v1), 2025-04-14

(93) [See or Recall: A Sanity Check for the Role of Vision in Solving Visualization Question Answer Tasks with Multimodal LLMs](http://arxiv.org/abs/2504.09809v1), 2025-04-14

(94) [Training Small Reasoning LLMs with Cognitive Preference Alignment](http://arxiv.org/abs/2504.09802v1), 2025-04-14

(95) [ReadMe.LLM: A Framework to Help LLMs Understand Your Library](http://arxiv.org/abs/2504.09798v1), 2025-04-14

(96) [Reasoning Court: Combining Reasoning, Action, and Judgment for Multi-Hop Reasoning](http://arxiv.org/abs/2504.09781v1), 2025-04-14

(97) [Reasoning without Regret](http://arxiv.org/abs/2504.09777v1), 2025-04-14

(98) [An Investigation of Large Language Models and Their Vulnerabilities in Spam Detection](http://arxiv.org/abs/2504.09776v1), 2025-04-14

(99) [Understanding and Optimizing Multi-Stage AI Inference Pipelines](http://arxiv.org/abs/2504.09775v1), 2025-04-14

(100) [Two Heads are Better Than One: Test-time Scaling of Multi-agent Collaborative Reasoning](http://arxiv.org/abs/2504.09772v1), 2025-04-14
